sentry_sdk/integrations/django/__init__.py

```diff
@@ -578,7 +580,10 @@ def execute(self, sql, params=None):
 
         with record_sql_queries(
             hub, self.cursor, sql, params, paramstyle="format", executemany=False
-        ):
+        ) as span:
+            vendor = connection.vendor
```

Comment:
I realized this strategy doesn't work if you have multiple connections open, but not sure how common that is.

Reply:
In Django it is common to have one connection to a read only replica of the database and one you use to change the data, so probably on bigger installations quite common.